scripts/node/validateTests.js

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env node
+
+/**
+ * Tupaia Config Server
+ * Copyright (c) 2019 Beyond Essential Systems Pty Ltd
+ */
+
+const path = require('path');
+
+const { executeCommand, getLoggerInstance } = require('./utilities');
+
+/**
+ * @typedef {Array<{ fileName, text }>} ExclusiveTest
+ */
+
+const PACKAGES = ['meditrak-server', 'web-config-server'];
+const MOCHA_TEST_BLOCKS = ['it', 'describe', 'context', 'specify'];
+const STD_ERROR_CODE = 1;
+
+const getTestDirectory = packageName =>
+  path.resolve(`${__dirname}/../../packages/${packageName}/src/tests`);
+
+const findExclusiveTests = async packageName => {
```

Comment:
This file is based on the now deleted `validateNoExclusiveTestsExist.js`, with a few changes... Unfortunately it's not easy to see in the diff here.

Also, it should use the `Scripts` class (defined in the same folder), but I didn't do this for ease of reviewing (and to avoid spending too much time)... Happy to do that in a following review.

Finally, we could consider using a dedicate cli creation library. [yargs](https://www.npmjs.com/package/yargs), [commander](https://www.npmjs.com/package/commander) seem popular choices. I'm not familiar with any of those, but they can probably assist as in creating elegant node scripts in the future!